Bio/SearchIO/_legacy/NCBIStandalone.py

```diff
@@ -25,9 +25,11 @@
 
 import sys
 import re
+from io import StringIO
```

Comment:
This is now a duplicate import, was already there two lines down - one of the things flake8 spotted:

```
$ flake8
./Bio/File.py:153:1: D205 1 blank line required between summary line and description
./Bio/SearchIO/_legacy/NCBIStandalone.py:30:1: F811 redefinition of unused 'StringIO' from line 28
```